app/views/tasks/edit.html.erb

```diff
@@ -5,6 +5,12 @@
 <div class="task edit">
   <h2>Edit task</h2>
```

Comment:
When I'm creating a new task I also see the `Edit task` title. 
Maybe it will be better to have two different pages that use the same form? (using `render partial`)
Also, if I leave the `Description` field or the `Deadline` empty I'm getting an error: `undefined method `strftime' for nil:NilClass`.